src/couch_prometheus/test/eunit/couch_prometheus_e2e_tests.erl

```diff
@@ -81,6 +81,7 @@ setup_prometheus(WithAdditionalPort) ->
         WithAdditionalPort,
         Persist
     ),
+    ok = config:set("prometheus", "port", ?PROM_PORT),
```

Comment:
Should we set `Persist=false`? Or we specifically want to set it and keep at in the config file across restarts here?

Reply:
Ah yes of course. Thanks for catching that!  Fixed in https://github.com/apache/couchdb/commit/ef2fcbc1f22d8e98ed76595efc155e4eca3f9467